docs/develop/node/intro/what-is-a-node.md

```diff
@@ -0,0 +1,23 @@
+---
+id: what-is-a-node
+title: What is a Node
+sidebar_label: What is a Node
+description: What is a NEAR Node and Why Run a Node
+---
+
+NEAR Protocol runs on a collection of publicly maintained computers (or "nodes"). All nodes are running the same `nearcore` codebase with the latest releases available on [Github](https://github.com/near/nearcore/releases/). In the following section, we will introduce three types of nodes.
+
+It is important to keep in mind all nodes runs the same codebase, but with different configurations. As such we have split up the instructions for running different nodes into different sections.
+
+## Why run a Node?
+
+You may decide to run a node of your own for a few reasons:
+
+- To join a network as a validator running a "validator node". Running a validator node is a public good and you are effectively securing the NEAR network and earning rewards.
+- To develop and deploy contracts on a node connected to `MainNet`, `TestNet` or `BetaNet` (†)
```

Comment:
I don't see it to be a good reason to run your own node. I think it is better to say something along these lines:

> To run applications that heavily depend on RPC performance and/or availability.

Reply:
> The cost ranges from 0.10-0.35 cents depending on tyoe of SSDs which are used across providers. For 1.5TB the cost would be somewhere 150-500 dollars depending on setup.

Yes!
@chefsale  May I please have some guidance and ballpark cost estimation numbers on hardware costs:

Validator node cost estimate: https://docs.near.org/docs/develop/node/validator/hardware
RPC node cost estimate: https://docs.near.org/docs/develop/node/rpc/hardware-rpc
Archival node cost estimate: https://docs.near.org/docs/develop/node/archival/hardware-archival - Is the storage cost of $150-$500 a monthly figure? 

Thanks!